client/src/components/App.jsx

```diff
@@ -60,17 +59,21 @@ class App extends React.Component {
       Math.round(rating[category] /= reviews.length);
     });
 
-    return rating;
+    this.setState({
+      reviews,
+      rating,
+    });
   }
 
   render() {
     const { reviews } = this.state;
```

Comment:
you can just write this as const { reviews, ratings } instead of two lines

Reply:
oh good suggestion. I have made the change!